revolut/__init__.py

```diff
@@ -215,15 +215,36 @@ def _delete(self, path, data=None):
         return self._request(requests.delete, path, data or {})
 
     def create_order(
-        self, amount=None, currency=None
+        self, amount, currency, token
     ):
         reqdata = {}
         if amount:
             reqdata["amount"] = amount
         if currency:
             reqdata["currency"] = currency
+        if token:
+            reqdata["merchant_order_ext_ref"] = token
         data = self._post("orders", data=reqdata or None)
-        return data
+        return Order(client=self, **data)
+
+    def orders(
+        self, from_date=None, to_date=None
+    ):
+        orders = []
+        reqdata = {}
+        if from_date:
+            reqdata["from_created_date"] = datetime.strptime(str(from_date), "yyyy-MM-dd'T'HH:mm:ssZ")
```

Comment:
Czy `dateutil.parser.parse` nie zadziała wprost? Mamy to już zaimportowane i używamy z powodzeniem.